packages/services/api/src/modules/shared/providers/storage.ts

```diff
@@ -69,7 +69,6 @@ export interface Storage {
   isReady(): Promise<boolean>;
   ensureUserExists(_: {
     superTokensUserId: string;
-    externalAuthUserId?: string | null;
```

Comment:
We do not use the externalAuthUserId anywhere for a long time now.

Reply:
This was used with Auth0, it's legacy, and can be removed for sure.